types/three/examples/jsm/objects/ReflectorForSSRPass.d.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    Mesh,
+    ShaderMaterial,
+    WebGLRenderTarget,
+    BufferGeometry,
+    WebGLRenderer,
+    Scene,
+    Camera,
+} from '../../../src/Three';
+
+export interface ReflectorShader {
+    defines: {
+        isDistanceAttenuation: boolean;
+        isFresnel: boolean;
+    };
+    uniforms: {
+        [key: string]: any;
+    };
```

Comment:
Can we improve this type? Maybe if we don't care about the specific properties, we can replace `any` with `Uniform<unknown>`.